packages/paste-core/components/combobox/stories/customization.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import {Stack} from '@twilio-paste/stack';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {Card} from '@twilio-paste/card';
+import {Text} from '@twilio-paste/text';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {Theme} from '@twilio-paste/theme';
+import {Combobox} from '../src';
+
+const items = ['a', 'b', 'c'];
+
+interface GroupedItem {
+  letter: string;
+  number: number;
+}
+
+const groupedItems = [
+  {letter: 'a', number: 1},
+  {letter: 'a', number: 2},
+  {letter: 'b', number: 3},
+  {letter: 'b', number: 4},
+  {letter: 'b', number: 5},
+  {letter: 'c', number: 6},
+  {letter: 'd', number: 7},
+];
+
+const ShowCustomization: React.FC<React.ReactNode> = ({children}): React.ReactElement => (
+  <Stack orientation="vertical" spacing="space50">
+    <Card>
+      <Theme.Provider>
+        {React.cloneElement(children as React.ReactElement, {'data-testid': 'non-customized'})}
+      </Theme.Provider>
+    </Card>
+    <Card>
+      <CustomizationProvider
+        baseTheme="default"
```

Comment:
Can we instead use the current theme here? @SiTaggart suggested this to me a while back and I think it's a good idea.

So before the return statement:

```
const ShowCustomization: React.FC<React.ReactNode> = ({children}): React.ReactElement => {
    const currentTheme = useTheme();

    return (
        <CustomizationProvider theme={currentTheme} />
    );
}

Reply:
Just made this switch, but wondering why exactly it's more accurate?

Reply:
It's for the side by side in Chromatic. If you use a hard coded base theme around your story, when chromatic snapshots it side by side with the dark theme your story will be in default for both the default and dark mode snapshots. 

If you detect and provide the current theme, you inherit the theme from the side by side

Reply:
Ok makes sense